### DISCUSSION TITLE
Login popup can't be clicked when app is in TestFlight.

### DISCUSSION THREAD

**Discussion by gfant** [Web3Auth/Web3Auth#478] [Help]
I coded an app in Flutter that uses web3auth to access the rest of the app. When I use the iOS simulator it works smoothly without any issue. But now I have released the first beta version in TestFlight and can't click on the popup that asks me to enter the web3auth website to login. Am I missing something for the TestFlight app?

**Comment by gfant** [comment]
This is the popup that I refer to
![telegram-cloud-photo-size-1-5492141043114682317-x](https://user-images.githubusercontent.com/61362029/186603879-171a1b46-6285-4050-9bb6-14e97ef72cea.jpg)

**Reply by chaitanyapotti** [reply]
@dhruvTor

**Reply by chaitanyapotti** [reply]
we just released a new version 1.0.0 for web3auth flutter sdk. Please update and let us know if you still face any issues

**Reply by dhruvTor** [reply]
can you add me on test-flight for this app my email is dhruv@tor.us

**Reply by gfant** [reply]
I can't :(

**Reply by yashovardhan** [reply]
@iam-agf is there any way you can replicate this in a sample app and share with us your code? We will try to debug it and get back to you

**Reply by gfant** [reply]
Sorry! I solved it. It was because of the version of `web3auth_flutter` . When you use a personal clone of the repo or a previous version it crashes! :D

**Comment by gfant** [comment]
https://stackoverflow.com/questions/73549793/share-your-information-dialog-disappear-in-ios-device
